packages/api/src/v2/types.ts

```diff
@@ -17,9 +17,9 @@ export interface Context {
   userInteraction: UserInteraction;
   logProvider: LogProvider;
   telemetryReporter: TelemetryReporter;
-  cryptoProvider: CryptoProvider;
+  cryptoProvider?: CryptoProvider;
```

Comment:
Is there any strong reason to make cryptoProvider and permissionRequestProvideroptional?